tensorboard/backend/application.py

```diff
@@ -190,8 +190,7 @@ def TensorBoardWSGIApp(
 
   Args:
     flags: An argparse.Namespace containing TensorBoard CLI flags.
-    plugins: A list of plugins, which can be provided as TBPlugin subclasses
-        or TBLoader instances or subclasses.
+    plugins: A list of plugin loader instances.
```

Comment:
Technically, old comment was correct. The plugin can be either TBPlugin or TBLoader.

Reply:
That's what I thought at first, but if I'm reading the code correctly:
If we did pass in a TBPlugin instance, things would break.  This method calls `make_plugin_loader()` on all the plugins passed in, which will throw a TypeError, since it only accepts TBLoader instances, or TBPlugin/TBLoader subclasses.

Reply:
It's correct that this can't be a TBPlugin instance, but it can be any of the three things the docstring originally listed (TBPlugin subclass, TBLoader subclass, or TBLoader instance) so unless I'm missing something, I don't think we actually wanted to change the wording?

Reply:
Sorry, I probably should have waited before merging too soon.

The docstring originally lists those 3, but the only type being passed in is `TBLoader` instances.  `TensorBoardWSGIApp` only has one caller, which provides a list that has already gone through `make_plugin_loader`.

`application.py` is not supposed to be publicly exposed, so if we assume that we are the only ones using TBWSGIApp, updating the comment is just aligning it to the existing narrow case.